### PR TITLE
aria2: appletls is not available on 10.6 or earlier

### PR DESCRIPTION
**Pull request by ken-cunningham-webuse**

### CODE REVIEW

File: net/aria2/Portfile [hunk 1]
```diff
@@ -35,3 +35,10 @@ configure.args      --with-gnutls --with-libgnutls-prefix=${prefix} \
                     --with-libintl-prefix=${prefix} \
                     --with-xml-prefix=${prefix} \
                     --without-sqlite3
+
+# appletls not available on 10.6 or earlier
+platform darwin {
+    if { ${os.major} <= 10 } {
```
Comment (larryv) [inline net/aria2/Portfile]: You should just use
```tcl
if {${os.platform} eq "darwin" && ${os.major} <= 10} {
```
and avoid the excess nesting.

Comment (ken-cunningham-webuse) [inline net/aria2/Portfile]: I like that! Thanks.

**Comment by ken-cunningham-webuse** [comment]
Just wanted to be clear though -- I'm not going to change this pull request. Please commit it as it is and I'll consider larry's changes for the next portfile fix I suggest. -- Best, Ken

**Comment by neverpanic** [comment]
While I don't object merging this as-is, isn't the point of having pull request reviews that the comments can be fixed before the code is merged?

**Comment by larryv** [comment]
Uh, yes. Yes it is. @ken-cunningham-webuse, why don’t you want to make the change?

**Comment by Schamschula** [comment]
I agree with @neverpanic. Please add @larryv's change.

**Comment by ken-cunningham-webuse** [comment]
It's a minor style change. It has no functional use. It makes the code (to me) less readable. It takes more time to do for no significant benefit. It's arbitrary. We could endlessly discuss style issues and get nothing done.

But OK. I found a reference how to do this:

<http://stackoverflow.com/questions/16748115/how-to-modify-github-pull-request>

and I might as well know how for the future. -- Best, Ken

**Comment by larryv** [comment]
Aesthetics are up to the maintainer, who is @Schamschula.

You just have to modify your branch; GitHub will pick up the new commits automatically.

**Comment by ken-cunningham-webuse** [comment]
I think that actually worked...

**Comment by Schamschula** [comment]
Looks good.

**Comment by raimue** [comment]
This is an example of a pull request that should have been squashed before merging. As a separate commit, without any connection to this pull request, the commit message does not make much sense.

**Pull request merged**